hal/src/main/native/athena/AnalogGyro.cpp

```diff
@@ -169,6 +169,7 @@ void HAL_CalibrateAnalogGyro(HAL_GyroHandle handle, int32_t* status) {
 
   HAL_InitAccumulator(gyro->handle, status);
   if (*status != 0) return;
+  std::printf("Callibrating analog gyro for %d seconds.", kCalibrationSampleTime);
```

Comment:
Use wpi::outs() instead of std::printf() and include a `\n` at the end.

Reply:
thanks.